lib/apiservers/engine/backends/container.go

```diff
@@ -1431,3 +1473,72 @@ func copyEscapable(dst io.Writer, src io.ReadCloser, keys []byte) (written int64
 func clientFriendlyContainerName(name string) string {
 	return fmt.Sprintf("/%s", name)
 }
+
+func processVolumeParam(volString string) (volumeFields, error) {
```

Comment:
Nit. I know there's a convention, but a 1500 line file is a bit long.  Please consider moving these `*Volume*` functions to their own file in this package.  `container_volumes.go` or something like that.  Your call.